bindings/java/devdoc/java_out_of_process_hld.md

```diff
@@ -0,0 +1,122 @@
+Java Out of Process Module API
```

Comment:
That would be true of C as well, but it allows a generic transformation of that JSON into a structure if the module were to need it for some reason or another. I don't quite understand the value either, but it's part of our API. @damonbarry or @darobs can you shine some light on this?